tests/regression/1_run_TA_API.sh

```diff
@@ -1,19 +1,47 @@
-make
+#!/bin/bash
+
+# Build options
+OPTIONS=(
+    ""
+    "--proxy_passthrough"
+    "--define mqtt=enable"
+    "--define db=enable"
+    "--define build_type=debug"
+    "--define build_type=profiling"
+)
+success=()
+fail=()
 
+# Install and start prerequisites
+make
+make MQTT
```

Comment:
`MQTT` should not be one of the available targets. Instead, it is just an option to enable certain features.

Reply:
`make MQTT` would build the third party MQTT library. I think it would be better that we do the regression tests with only the necessary third party libraries.